### PR TITLE
Fixes bug 1195307 - Added first appearance date to new topcrashers.

### PR DESCRIPTION
**Pull request by AdrianGaudebert**
@peterbe r?

### CODE REVIEW

File: webapp-django/crashstats/topcrashers/tests/test_views.py [hunk 1]
```diff
@@ -16,9 +16,10 @@ class TestViews(BaseTestViews):
     base_url = reverse('topcrashers:topcrashers')
 
     @mock.patch('crashstats.crashstats.models.Bugs.get')
-    def test_topcrashers(self, rpost):
+    @mock.patch('requests.post')
+    def test_topcrashers(self, rpost, bugs_get):
 
-        def mocked_post(**options):
+        def mocked_bugs(**options):
```
Comment (peterbe) [inline webapp-django/crashstats/topcrashers/tests/test_views.py]: great rename! thanks

File: webapp-django/crashstats/topcrashers/views.py [hunk 2]
```diff
@@ -305,11 +317,22 @@ def topcrashers(request, days=None, possible_days=None, default_context=None):
...
             else:
                 crash['bugs'] = bugs[sig]
+
+        # Augment with first appearance dates.
+        if sig in sig_date_data:
+            first_date = isodate.parse_datetime(sig_date_data[sig])
+            crash['first_report'] = first_date.strftime('%Y-%m-%d')
+            crash['first_report_exact'] = first_date.strftime(
+                '%Y-%m-%d %H:%M:%S'
+            )
```
Comment (peterbe) [inline webapp-django/crashstats/topcrashers/views.py]: Date formatting belongs in templates. I'd much prefer if you do `crash['first_report'] = first_date` and then in the template you format it nicely. 

Also, what about timezone? Do we not get any?

**Comment by peterbe** [comment]
What is it with you and getting unit test errors in travis these day :)

**Comment by AdrianGaudebert** [comment]
@peterbe comments addressed!

**Comment by peterbe** [comment]
r+

**Pull request merged**